app/views/async_info/navigation_links.html.erb

```diff
@@ -1,2 +1,4 @@
-<%= render partial: "shared/auth_widget", locals: { tracking_id: "ca_hamburger_home_page", source: "mobile_hamburger_nav" } unless user_signed_in? %>
-<%= render partial: "layouts/main_nav", locals: { context: "hamburger" } %>
+<% cache(release_adjusted_cache_key("hamburger-menu--#{user_signed_in?}"), expires_in: 15.minutes) do %>
```

Comment:
Added some Fragment caching for the hamburger menu. Its useful to note that this is also being done in https://github.com/forem/forem/blob/b03839ba9f6ace4f9ead3a656e69c0fae30c73e5/app/views/articles/_sidebar.html.erb#L3 for the main_nav etc.